src/luxor_overrides.jl

```diff
@@ -273,3 +274,298 @@ function background(background_color)
     end
     Luxor.background(background_color)
 end
+
+function apply_transform(transform::Vector{Float64}, poly::Vector{Point})
```

Comment:
We aren't strict with this atm but I would like to avoid having methods in this file that are not actually overrides. Maybe we can have a new file like `luxor_overrides_utils.jl` or something like that.